features/step_definitions/table_steps.rb

```diff
@@ -40,7 +40,7 @@
 
   retries ||= 1
 
-  expected.diff! table
+  expected.diff! table, { surplus_col: true }
```

Comment:
The comparison will fail if the actual table has an extra column that is not defined in the expected table.